pages/search/advanced.tsx

```diff
@@ -173,13 +168,13 @@ export default function AdvancedSearch({
   }
 
   useEffect(() => {
-    const { from, to, range } = dateFilterProps.dateError || {}
+    const { from, to, range } = dateError || {}
     if (alert && !from && !to && !range && notificationRef.current) {
       setTimeout(() => {
```

Comment:
feel free to ignore if out of scope, but i'm wondering if we at least want to flag this to be updated with the new focus handler we worked on a few months ago?